themes/wilson_theme_starterkit/README.md

```diff
@@ -92,6 +92,14 @@ npm run watch:scripts
 npm run dev
 ```
 
+### Automated testing
+
+The theme has an integration with [Cypress](https://www.cypress.io/), which allows front end tests to be written. Tests should be added as JS files within `./cypress/integration` and will be picked up by the Cypress Test Runner. More information on writing testing can be found [here](https://docs.cypress.io/guides/getting-started/writing-your-first-test).
+```bash
+npm run test:local
+```
+This runs Cypress tests (headlessly) to completion, using a base URL of `https://wilson-test.ddev.site`. Note - this base URL will need to be updated to the relevant project url once this theme has been installed.
```

Comment:
Does renaming the theme happen manually when it's installed? Or is it automated? 

If it's automated, maybe that can update the Wilson test URL too? Or use environment variables or something.